packages/backend/migrations/20230918113102_rename_payment_pointer_tables.js

```diff
@@ -128,7 +128,7 @@ exports.down = function (knex) {
       'ALTER INDEX "walletAddressKeys_pkey" RENAME TO "paymentPointerKeys_pkey"'
     ),
     knex.raw(
-      'ALTER TABLE "paymentpointerKeys" DROP CONSTRAINT "walletaddresskeys_paymentpointerid_foreign"'
+      'ALTER TABLE "paymentPointerKeys" DROP CONSTRAINT "walletaddresskeys_walletaddressid_foreign"'
```

Comment:
What is `walletaddresskeys_walletaddressid_foreign`?

Reply:
It's a foreign key on the table `walletAddressKeys` that links it to `walletAddress` via its `id` property. The line referenced here is the rollback for this migration, so it's part of reverting the foreign key from referring to wallet addresses to payment pointers.